pkg/util/docker.go

```diff
@@ -3,22 +3,25 @@ package util
 import (
 	"encoding/json"
 	"io"
+	"os"
 
 	"github.com/docker/docker/pkg/jsonmessage"
 )
 
-// PipeDockerOutput pipes a reader that spits out jsonmessage structs into a
-// writer, usually stdout. It returns normally when the reader is exhausted, or
-// in error if one occurs.
-func PipeDockerOutput(r io.ReadCloser, w io.Writer) error {
+// PipeDockerOutput pipes a reader that spits out jsonmessage structs into one
+// or many writer, usually stdout. It returns normally when the reader is exhausted,
+// or in error if one occurs.
+func PipeDockerOutput(r io.ReadCloser, writers ...io.Writer) error {
```

Comment:
I'd say accept a single `Writer`, and have the caller pass a [`MultiWriter`](https://golang.org/pkg/io/#MultiWriter) if they want to tee outputs.

Reply:
Not needed anymore since I don't need to use this function anymore. We can change it in the future if needed!